src/libcore/time.rs

```diff
@@ -23,6 +23,22 @@ const MILLIS_PER_SEC: u64 = 1_000;
 const MICROS_PER_SEC: u64 = 1_000_000;
 const MAX_NANOS_F64: f64 = ((u64::MAX as u128 + 1)*(NANOS_PER_SEC as u128)) as f64;
 
+/// The duration of one second.
+#[unstable(feature = "duration_constants", issue = "0")]
```

Comment:
The issue number should be updated for the tracking issue.

Reply:
Ah, good call. @stjepang, can you please update those to reference the tracking issue?

Reply:
Done.